druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/server/lookup/namespace/KafkaManager.java

```diff
@@ -54,13 +59,20 @@
 
     private final ProtobufSchemaFactory protobufSchemaFactory;
 
+    private KafkaProducer<String, byte[]> kafkaProducer;
+
     @Inject
     public KafkaManager(Provider<MahaExtractionCacheManager> namespaceExtractionCacheManager,
                         @Named("kafkaProperties") final Properties kafkaProperties,
                         ProtobufSchemaFactory protobufSchemaFactory) {
         this.kafkaProperties.putAll(kafkaProperties);
         this.namespaceExtractionCacheManager = namespaceExtractionCacheManager;
         this.protobufSchemaFactory = protobufSchemaFactory;
+        final Properties properties = new Properties();
+        properties.putAll(kafkaProperties);
+        properties.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class.getName());
+        properties.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, ByteArraySerializer.class.getName());
+        this.kafkaProducer = new KafkaProducer<>(properties);
```

Comment:
Probably should configure the producer to drop messages if the underlying message cannot be written, we don't want to queue / batch the messages beyond a max value.  A failed write to kafka may just enqueue the message for the send thread and we don't want it to grow unbounded, and we don't want it to block or through exceptions.  It should drop messages if it can't write after queue is full.